### PR TITLE
Install buildifier and run it in check_formatting

### LINKED ISSUE #130: Check BUILD files format with buildifier
See https://github.com/bazelbuild/buildtools/releases

### PR DESCRIPTION
**Pull request by remyabel**
Fixes #130

### CODE REVIEW

File: scripts/check_formatting [hunk 1]
```diff
@@ -3,6 +3,10 @@
 set -o errexit
 set -o nounset
 
+
+# Apply buildifier to fix the BUILD and .bzl formatting.
+find oaks examples toolchain -type f -exec buildifier -mode=fix {} +
```
Comment (tiziano88) [inline scripts/check_formatting]: s/oaks/oak/?

Comment (remyabel) [reply scripts/check_formatting]: Thanks.

File: scripts/check_formatting [hunk 2]
```diff
@@ -3,6 +3,10 @@
 set -o errexit
 set -o nounset
 
+
+# Apply buildifier to fix the BUILD and .bzl formatting.
+find oak examples toolchain -type f -exec buildifier -mode=fix {} +
```
Comment (tiziano88) [inline scripts/check_formatting]: This script is supposed to just check formatting, but from this command line I'm guessing buildifier is actually formatting files if necessary, which is not what we want.

Comment (remyabel) [reply scripts/check_formatting]: Do you prefer check mode or print_if_changed mode? Check mode only shows files that need to be changd, while print_if_changed will show the diff.

**Comment by remyabel** [comment]
> I'm surprised there were no other fixes! Are we really that good at writing build files by hand? :)

If you want it to be pedantic, we can enable `-lint warn`, but it's off by default.

File: scripts/check_formatting (hunk 2 above)
Comment (remyabel) [reply scripts/check_formatting]: Nevermind, I see now that you have a separate file for actually applying the formatting.

**Pull request merged**